tokenlists/_cli.py

```diff
@@ -15,7 +15,7 @@ def choices(self):
         return list(TokenListManager().available_tokenlists())
 
 
-@click.group()
+@click.group(short_help="Work with token lists")
```

Comment:
Made a short help here because the help text in the doc str was always with elipsis